README.md

```diff
@@ -52,6 +52,15 @@ Install the following extensions:
 
 ## Code scaffolding
 
+## Commit format
+  We are using format by release and tags: https://semver.org/ 
```

Comment:
A commit message needs to start with one of the following words to bump the application version properly (This application is following a semver strategy for versioning https://semver.org/)